src/components/existing-request/nr-approved-gray-box.vue

```diff
@@ -258,6 +251,26 @@ export default class NrApprovedGrayBox extends Mixins(CommonMixin) {
       this.isApprovedOrConsentUnRequired
     )
   }
+
+  async amalgamateNowClicked () {
+    const legalType = this.entityTypeToCorpType(this.getEntityTypeCd)
+    if (this.isAuthenticated) {
+      if (this.isSupportedAmalgamation) {
+        console.log('1')
+        await this.amalgamateNow(legalType)
+      } else {
+        console.log('2')
```

Comment:
Eventually, delete the debugging statements.

By the way, did you know that you can debug this directly in VS Code?